src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -409,6 +411,10 @@ public function has($id)
      */
     public function get($id, $invalidBehavior = ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE)
     {
+        if (!$this->compiled) {
+            @trigger_error(sprintf('Calling %s before compiling the container is deprecated since version 3.2 and will throw an exception in 4.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
This error message will be displayed like this:

```
Calling get before compiling the container is ...
```

`get` here may be confusing. What if we add the `(` and `)` to the method name?

```
Calling get() before compiling the container is ...
```

Or wrap the method name with quotes:

```
Calling "get" before compiling the container is ...
```